test/powershell/Modules/Microsoft.PowerShell.Core/Get-Command.Tests.ps1

```diff
@@ -269,4 +269,9 @@ Describe "Get-Command Tests" -Tags "CI" {
         $result.Count | Should -Be 2
         $result.Name | Should -Be "Add-Content","Get-Content"
     }
+
+    It "Excluding modules works" {
```

Comment:
You've made some changes to command completion, can you add tests to https://github.com/PowerShell/PowerShell/blob/master/test/powershell/Host/TabCompletion/TabCompletion.Tests.ps1

Reply:
Test added.